assets/src/dashboard/components/cardGridItem/cardPreview.js

```diff
@@ -31,8 +31,10 @@ import { ActionLabel } from './types';
 
 const PreviewPane = styled.div`
   position: relative;
-  border-radius: 8px;
+  border-radius: 4px;
```

Comment:
Noice!  But I think the `EditControls` component right below this one (on line 54) needs to be 4px too... 😱

Reply:
Screenshot to show how it's looking (on hover), notice the small gap between the gray and the card:
![image](https://user-images.githubusercontent.com/40646372/82103236-1f89bc00-96c7-11ea-98a4-8337ec932ba5.png)